isthmus/src/main/java/io/substrait/isthmus/SqlToSubstrait.java

```diff
@@ -39,19 +64,39 @@ public Plan execute(String sql, Function<List<String>, NamedStruct> tableLookup)
   }
 
   public Plan execute(String sql, List<String> tables) throws SqlParseException {
-    var pair = registerCreateTables(tables);
-    return executeInner(sql, factory, pair.left, pair.right);
+    var result = registerCreateTables(tables);
+    return executeInner(sql, factory, result.validator(), result.catalogReader());
```

Comment:
@davisusanibar what is the motivation for this change? I feel like the previous was okay, but would like to know your thoughts.

Reply:
I just rolled back.